flight/maestro/maestro.py

```diff
@@ -0,0 +1,287 @@
+"""
+File containing the Maestro class to be able to control the Maestro board
+"""
+
+import time
+import serial
+
+
+class Maestro:
+    """
+    Maestro class to control the Maestro board
+
+    Attributes
+    ----------
+    serial: serial.Serial
+        Serial object to be able to read and send messages to and from the board.
+
+    command_prefix: str
+        The command protocol prefix that is sent before every command to the board.
+
+    targets: list(int)
+        A list of the current targets for each channel.
```

Comment:
Shouldn't have extra newlines between these.
```suggestion
    serial: serial.Serial
        Serial object to be able to read and send messages to and from the board.
    command_prefix: str
        The command protocol prefix that is sent before every command to the board.
    targets: list(int)
        A list of the current targets for each channel.
```